memory_app_fastflash.ld

```diff
@@ -0,0 +1,9 @@
+/* STM32F205RE - 512K Flash, 128K RAM */
+/* program starts at 0x2001200 */
+MEMORY
+{
+	rom (rx) : ORIGIN = 0x20012000, LENGTH = 40K
```

Comment:
Currently rom and ram are overlapping.  Should be either moved to 0x20016000 or extended to 56K.